p2pserver/peer/peer.go

```diff
@@ -395,8 +373,8 @@ func (this *Peer) UpdateInfo(t time.Time, version uint32, services uint64,
 	this.SetHeight(uint64(height))
 }
 
-//parseIPAddr return ip address
-func parseIPAddr(s string) (string, error) {
+//ParseIPAddr return ip address
+func (this *Peer) ParseIPAddr(s string) (string, error) {
```

Comment:
this method has nothing to do with Peer,  make it a free helper function for easier review.